diff -up vendor/apereo/phpcas/source/CAS/Client.php.patch vendor/apereo/phpcas/source/CAS/Client.php
--- vendor/apereo/phpcas/source/CAS/Client.php.patch	2025-08-20 08:44:55.130255365 +0200
+++ vendor/apereo/phpcas/source/CAS/Client.php	2025-08-20 09:08:47.140393998 +0200
@@ -788,7 +788,7 @@ class CAS_Client
             'file' => $dbg[1]['file'],
             'line' => $dbg[1]['line'],
             'method' => $dbg[1]['class'] . '::' . $dbg[1]['function'],
-            'result' => (boolean)$auth
+            'result' => (bool)$auth
         );
     }
     private $_authentication_caller;
@@ -3166,7 +3166,7 @@ class CAS_Client
                 $proxiedService->setCasClient($this);
             }
             return $proxiedService;
-        case PHPCAS_PROXIED_SERVICE_IMAP;
+        case PHPCAS_PROXIED_SERVICE_IMAP:
             $proxiedService = new CAS_ProxiedService_Imap($this->_getUser());
             if ($proxiedService instanceof CAS_ProxiedService_Testable) {
                 $proxiedService->setCasClient($this);
